modules/tinymce/src/core/main/ts/api/OptionTypes.ts

```diff
@@ -197,6 +197,8 @@ interface BaseEditorOptions {
   text_patterns?: RawPattern[] | false;
   theme?: string | ThemeInitFunc | false;
   theme_url?: string;
+  model?: string;
+  model_url?: string;
```

Comment:
These aren't in the right place since the list is in alphabetical order. It's fixed in the next PR though so up to you if you want to fix that here.

Reply:
let's not complicate these PRs more than they already are :)

Reply:
Ah, missed that this files had changed when cherry-picking commits. I'll fix that up. Thanks

Reply:
oh man GitHub pending comments are weird, aren't they? now our comments are out of order.

Reply:
To save having to run another build, I won't make the change for now